saleor/plugins/avatax/__init__.py

```diff
@@ -449,7 +455,9 @@ def get_cached_tax_codes_or_fetch(
     tax_codes = cache.get(TAX_CODES_CACHE_KEY, {})
     if not tax_codes:
         tax_codes_url = urljoin(get_api_url(config.use_sandbox), "definitions/taxcodes")
-        response = api_get_request(tax_codes_url, config)
+        response = api_get_request(
+            tax_codes_url, config.username_or_account, config.password_or_license
+        )
         if response and "error" not in response:
```

Comment:
You raise the TaxError when response has error field so checking `"error" not in response` is not needed, right?